src/app/session/login/login.component.html

```diff
@@ -39,12 +39,11 @@
       <mat-error *ngIf="session.getAuthError()">{{
         session.getAuthError()?.error?.error?.message || session.getAuthError()?.statusText || session.getAuthError()?.message
       }}</mat-error>
-
-      <button class="login-button" mat-raised-button color="primary" (click)="login()">Login</button>
+      <stack [spacing]="5">
+        <button class="login-button" mat-raised-button color="primary" (click)="login()">Login</button>
+        <text-divider style="width: 100%">{{ i18n.get('or') }}</text-divider>
```

Comment:
Vielleicht könnte man in der TextDivider Component gleicht auf `:host` die width auf 100% setzen. (https://angular.io/guide/component-styles#host)